plugins/hydra_ray_launcher/tools/config.yaml

```diff
@@ -0,0 +1 @@
+cluster_name: default
```

Comment:
what is this file configuring?
it should have a more specific name. config.yaml is too generic.